lib/perl/Genome/WorkflowBuilder/Command.pm

```diff
@@ -40,6 +41,65 @@ sub create {
     return $self;
 }
 
+sub get_ptero_builder_task {
+    require Ptero::Builder::Detail::Workflow::Task;
+
+    my $self = shift;
+
+    $self->validate;
+
+    my %params = (
+        name => $self->name,
+        methods => [
+            $self->_get_ptero_shortcut_method,
+            $self->_get_ptero_execute_method,
+        ],
+    );
+    if (defined $self->parallel_by) {
+        $params{parallel_by} = $self->parallel_by;
+    }
+    return Ptero::Builder::Detail::Workflow::Task->new(%params);
+}
+
+sub _get_ptero_shortcut_method {
+    require Ptero::Builder::ShellCommand;
+
+    my $self = shift;
+    return Ptero::Builder::ShellCommand->new(
+        name => 'shortcut',
+        parameters => {
+            commandLine => [
+                'genome', 'ptero', 'wrapper', 'command',
+                '--command-class', $self->command,
+                '--method', 'shortcut',
+            ],
+            environment => \%ENV,
+            user => $ENV{USER},
```

Comment:
I don't know if it matters or if this is an exception but I think we usually use `Genome::Sys->username` instead of `$USER`.